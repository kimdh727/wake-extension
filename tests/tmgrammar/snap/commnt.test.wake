# Wake SUPPORT Only Line Comment

# This is Comment
This is not Comment
