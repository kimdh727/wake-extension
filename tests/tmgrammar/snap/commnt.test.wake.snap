># Wake SUPPORT Only Line Comment
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.wake comment.line
>
># This is Comment
#^^^^^^^^^^^^^^^^^ source.wake comment.line
>This is not Comment
#^^^^^^^^^^^^^^^^^^^^ source.wake
>